src/Ball.js

```diff
@@ -114,7 +119,7 @@ export class Ball {
             otherBall.state = STATES.incubating
             RUN.results[STATES.incubating]++
             RUN.results[STATES.well]--
-            if (this.isAwareToBeInfected(otherBall)) {
+            if (this.isAwareToBeInfected(otherBall) && this.willRespectAutoIsolation) {
```

Comment:
I think this check needs to be changed as it stops movements for `otherBall` when `this` ball will respect autoisolation.
```suggestion
            if (otherBall.isAwareToBeInfected(this) && otherBall.willRespectAutoIsolation) {
```

Reply:
Absolutely yes!

Reply:
Fixed with the last commit.